dex/ws/wslink.go

```diff
@@ -354,7 +354,7 @@ func (c *WSLink) outHandler(ctx context.Context) {
 		// need to drain it, just the outQueue so SendNow never hangs.
 
 		c.log.Tracef("Sent %d and dropped %d messages to %v before shutdown.",
-			writeCount, lostCount, c.ip)
+			writeCount, lostCount, c.addr)
 	}()
```

Comment:
I'm not seeing anything locally now. Doesn't output with loopback?
```
2020-12-18 16:19:42.443 [DBG] AUTH: Link down: id=2, ip=.
2020-12-18 16:19:42.443 [TRC] COMM[WS]: Sent 5 and dropped 0 messages to  before shutdown.
```

Reply:
Bug.  Going back in...

Reply:
Sorry, guys, I decided to rework this a little more thoroughly.

Logging is now as before:

dcrdex (client accessing)

```
[DBG] COMM: Starting websocket handler for 127.0.0.1:37068
[TRC] COMM: New websocket client 127.0.0.1
[TRC] COMM[WS]: Starting websocket messaging with peer 127.0.0.1
...
[TRC] COMM[WS]: Sent 11 and dropped 0 messages to 127.0.0.1 before shutdown.
[DBG] AUTH: Link down: id=0, ip=127.0.0.1.
[TRC] COMM: Disconnected websocket client 127.0.0.1
```

dexc (browser accessing)

```
[DBG] WEB[WS]: New websocket client 127.0.0.1:55064
[TRC] WEB[WS][127.0.0.1:55064]: Starting websocket messaging with peer 127.0.0.1:55064
...
[TRC] WEB[WS][127.0.0.1:55064]: Sent 4 and dropped 0 messages to 127.0.0.1:55064 before shutdown.
[TRC] WEB[WS]: Disconnected websocket client 127.0.0.1:55064
```

I changed `dex.IPKey` so the address is recoverable, and I limited its use to `server/comms.Server` where it is actually used for rate limiting and banishment.